addon/store.js

```diff
@@ -46,45 +46,39 @@ var Store = Ember.Object.extend({
       this.set("array", {});
     },
     clear(type) {
-        if(type === undefined) {
+        if (type === undefined) {
             this.reset();
         }
+
         delete this.get("identityMap")[type];
         arrayForType(type, this).clear();
-
-        run.once(() => {
```

Comment:
Correct, `run.once fn` needs to be a regular function otherwise it can be called multiple times within a run loop